app/src/main/res/values-qq/strings.xml

```diff
@@ -758,4 +758,6 @@
   <string name="text_input_dialog_ok_button_text">Positive action button text in pop-up dialog for text input.\n{{Identical|OK}}</string>
   <string name="text_input_dialog_cancel_button_text">Negative action button text in pop-up dialog for text input.\n{{Identical|Cancel}}</string>
   <string name="custom_date_picker_dialog_cancel_button_text">Negative action button text in date picker pop-up dialog.\n{{Identical|Cancel}}</string>
+  <string name="suggested_edits_add_translation">Label that prompts the user to translate a description of an article that does not yet have one.</string>
+  <string name="suggested_edits_edit_translation">Label that prompts the user to edit a translated description again.</string>
```

Comment:
Can these have better names like `suggested_edits_add_translation_button` and `suggested_edits_edit_translation_button`?

Reply:
Done